src/client/v2/algod/getBlockOffsetTimestamp.ts

```diff
@@ -0,0 +1,8 @@
+import JSONRequest from '../jsonrequest';
+
+export default class GetBlockOffsetTimestamp extends JSONRequest {
```

Comment:
I know these are relatively simple endpoints, but I highly recommend unpacking the responses into the generated model types for new endpoints. That way the user will see the response as a typed object (`GetBlockTimeStampOffsetResponse` in this case), instead of a generic `Record<string, any>` value.

To do this, you can inherit from `JSONRequest<ModelType, Record<string, any>>`, and implement the `prepare(body: Record<string, any>): ModelType` method. See how `GetApplicationBoxes` does this here: https://github.com/algorand/js-algorand-sdk/blob/a89046acaf9502f496fca9d781bf33047c969c01/src/client/v2/algod/getApplicationBoxes.ts#L57-L60

Really _all_ of our endpoints should do this, but for backwards compatibility we have not changed endpoints that existed before this behavior was available.

Reply:
Sounds good I'll make the change.

I'm also going to add an issue for that in #654 so that we can track these changes for the next time we want to make breaking changes.

Reply:
Issue for posterity: https://github.com/algorand/js-algorand-sdk/issues/771